rectangle.js

```diff
@@ -0,0 +1,35 @@
+var pry = require('pryjs');
+
+var area = function(rectangle) {
+  return rectangle.width * rectangle.height
+};
+
+var perimeter = function(rectangle) {
+  return (rectangle.width * 2) + (rectangle.height * 2)
+};
+
+var diagonal = function(rectangle) {
+  var width_squared = (rectangle.width * rectangle.width);
+  var height_squared = (rectangle.height * rectangle.height);
+  return Math.sqrt(width_squared + height_squared);
+};
+
+var diagonal = function(rectangle) {
```

Comment:
i'm not sure i'm seeing the difference between this and the function you defined on line 11. is there anything different between the two?